scene/audio/audio_stream_player.cpp

```diff
@@ -278,6 +278,18 @@ void AudioStreamPlayer::set_bus(const StringName &p_bus) {
 	AudioServer::get_singleton()->lock();
 	bus = p_bus;
 	AudioServer::get_singleton()->unlock();
+
+	bool found = false;
+	for (int i = 0; i < AudioServer::get_singleton()->get_bus_count(); i++) {
+		if (AudioServer::get_singleton()->get_bus_name(i) == bus) {
+			found = true;
+			break;
+		}
+	}
+
+	if (!found) {
+		WARN_PRINT("There is no bus '" + p_bus + "' defined.");
+	}
```

Comment:
I think this should be wrapped in `#ifdef DEBUG_ENABLED`.

Reply:
You're correct. Should it be also the same for `get_bus` - or are we leaving warning for all builds?

Reply:
Well, the warning is conditional and it doesn't affect correct code, so it could be left as is.

Reply:
No, I meant - `#ifdef DEBUG_ENABLED` - should that surround both methods' `WARN_PRINT` additions?

Reply:
And I meant that in `set_bus` whole new code should be wrapped, but `get_bus` doesn't need to xd

Reply:
Done as requested 😄